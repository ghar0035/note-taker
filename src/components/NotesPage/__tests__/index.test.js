import renderer from "react-test-renderer";
import NotePage from "../index.js";
import { MemoryRouter as Router } from "react-router-dom";

test("renders note page", () => {
    const tree = renderer
      .create(
        <Router>
          <NotePage />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });