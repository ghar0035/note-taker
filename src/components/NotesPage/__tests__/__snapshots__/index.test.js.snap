// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders note page 1`] = `
<div
  className="sc-gswNZR kVgipI"
>
  <img
    alt="no note found"
    src="no_notes.png"
    width="400"
  />
  <p>
    No notes is added
  </p>
  <button
    className="btn"
    onClick={[Function]}
  >
    Add Note
  </button>
</div>
`;
